app/views/hub/signup_selections/_signup_selection.html.erb

```diff
@@ -0,0 +1,13 @@
+<div>
+  <%= item.upload.filename.presence || "filename" %>
+  by <%= item.user&.email || "unknown user" %>:
+  <%= item.status.capitalize %>
```

Comment:
trying this locally - SignupSelections don't have a `status`

Reply:
Agreed :) creating that copy-pasta was the last thing I did before pushing this WIP commit; I'll probably look at that in a moment.